### DISCUSSION TITLE
Dúvidas sobre API's!

### DISCUSSION THREAD

**Discussion by SamuelPyerre** [balta-io/2811#20] [Q&A] [answered]
Bom dia, estou fazendo o Curso de Backend .NET e vi que o Balta ensina a criar API's com o Asp, mas sendo sincero eu não consigo compreender como eu juntaria uma API com o lado do cliente, ou seja, um frontend, como por exemplo em Angular. Digo isso pois vejo muitos professores ensinarem a criar sistemas web com asp, com frontend e backend no mesmo projeto, como sou leigo não sei se isso é algo já antigo, mas mesmo assim, gostaria de um help para que eu possa entender melhor. Para simplificar, queria entender como no final de um projeto real, como funciona essa questão do lado do cliente, as telas... Parece um pouco confuso minha dúvida, mas se precisarem de melhor esclarecimento, estou aqui. ;)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @SamuelPyerre, realmente é uma dúvida válida, mas a resposta é simples... eles independem... o front não depende do back... você pode criar um frontend em react, angular, vue, blazar... e pode criar o backend em asp.net, java, py, go, js...

Nossa carreira mostra como conectar eles:
https://balta.io/carreiras/desenvolvedor-backend-dotnet

👇
![Screen Shot 2022-05-10 at 08 48 10](https://user-images.githubusercontent.com/965305/167621498-44161660-3250-46f9-9454-c9b9c1367621.png)
